public/app/plugins/panel/geomap/layers/data/StyleEditor.tsx

```diff
@@ -108,7 +108,7 @@ export const StyleEditor: FC<StandardEditorProps<StyleConfig, StyleEditorOptions
             <InlineFieldRow>
               <InlineField label={'Symbol'}>
                 <ResourceDimensionEditor
-                  value={value.symbol ?? defaultStyleConfig.symbol}
+                  value={value?.symbol ?? defaultStyleConfig.symbol}
```

Comment:
It's a little strange to me that `value` prop would ever be undefined, not a big deal for getting these fixes in but I will investigate this as part of my refactor changes

Reply:
Yeah it's showing up in older panels so I think it's something to do with the shape of the config changing and/or when the value sets text, size, symbol etc.

We could also probably address this by handling with a migration case in the geomap migrations but it seemed like we were trying to minimize the number of migration cases were checking for.